### PR TITLE
Fix windows build by conforming line endings to Unix

### PR DESCRIPTION
**Pull request by aquarat**
This fixes the `airnode-deployer` build on Windows.

The problem was that Git on Windows, by default, swaps Unix end-of-line bytes with Windows _EOL_ bytes (LF vs CRLF), which makes `/entrypoint.sh` not executable (it breaks the shebang).

Ideally one would want to configure git to not do this, which is possible, but the process is inconsistent depending on which Git you're using (Git for Windows, Github Desktop, etc.) and seemingly has to be applied in the Dockerfile too. Using `sed`, a utility the Alpine image comes bundled with, to do the character removal, seems like an elegant one-line solution that doesn't noticeably increase the docker image size.

Git automatically converts files back to Unix when a Windows user does a commit and push.

### CODE REVIEW

**Review by Siegrift — COMMENTED** [review]
Those shebang lines are problem aslo for binaries - good job finding this out.

I will test this today evening and give approve if it works.

**Comment by aquarat** [comment]
> Those shebang lines are problem aslo for binaries - good job finding this out.
Thanks man
It shouldn't affect machine-code binaries, only text files (like .sh, .py, etc.). I'm hoping I didn't miss something?

**Comment by dcroote** [comment]
I was able to build on Windows cmd and use the image to successfully deploy, but the deployed airnode doesn't work for me. In the CloudWatch `dev-startCoordinator` log group, I get the following (I've tried an airnode-example and the docs quick deploy tutorial, and I've confirmed the configuration works with the pre-built `0.4.0` image)

```text
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
|   timestamp   |                                                                                                                                                                                                                                                                                                                                                                                                                 message                                                                                                                                                                                                                                                                                                                                                                                                                 |
|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| 1645166906038 | 2022-02-18T06:48:26.038Z undefined ERROR Uncaught Exception  {"errorType":"Runtime.HandlerNotFound","errorMessage":"index.startCoordinator is undefined or not exported","stack":["Runtime.HandlerNotFound: index.startCoordinator is undefined or not exported","    at Object.module.exports.load (/var/runtime/UserFunction.js:246:11)","    at Object.<anonymous> (/var/runtime/index.js:43:30)","    at Module._compile (internal/modules/cjs/loader.js:1085:14)","    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)","    at Module.load (internal/modules/cjs/loader.js:950:32)","    at Function.Module._load (internal/modules/cjs/loader.js:790:12)","    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)","    at internal/main/run_main_module.js:17:47"]}  |
| 1645166906043 | 2022-02-18T06:48:26.043Z undefined ERROR (node:9) Warning: Accessing non-existent property 'startCoordinator' of module exports inside circular dependency (Use `node --trace-warnings ...` to show where the warning was created)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
| 1645166906108 | START RequestId: dae0a9e6-ccb1-4f5b-b690-938884e48414 Version: $LATEST                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
| 1645166906478 | 2022-02-18T06:48:26.478Z undefined ERROR Uncaught Exception  {"errorType":"Runtime.HandlerNotFound","errorMessage":"index.startCoordinator is undefined or not exported","stack":["Runtime.HandlerNotFound: index.startCoordinator is undefined or not exported","    at Object.module.exports.load (/var/runtime/UserFunction.js:246:11)","    at Object.<anonymous> (/var/runtime/index.js:43:30)","    at Module._compile (internal/modules/cjs/loader.js:1085:14)","    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)","    at Module.load (internal/modules/cjs/loader.js:950:32)","    at Function.Module._load (internal/modules/cjs/loader.js:790:12)","    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)","    at internal/main/run_main_module.js:17:47"]}  |
| 1645166906500 | 2022-02-18T06:48:26.500Z undefined ERROR (node:9) Warning: Accessing non-existent property 'startCoordinator' of module exports inside circular dependency (Use `node --trace-warnings ...` to show where the warning was created)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
| 1645166906539 | END RequestId: dae0a9e6-ccb1-4f5b-b690-938884e48414                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
| 1645166906539 | REPORT RequestId: dae0a9e6-ccb1-4f5b-b690-938884e48414 Duration: 429.67 ms Billed Duration: 430 ms Memory Size: 512 MB Max Memory Used: 12 MB                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
| 1645166906539 | Unknown application error occurred Runtime.HandlerNotFound                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

**Comment by aquarat** [comment]
@dcroote 
Wow that's weird, thanks for checking, I'll investigate.

**Comment by aquarat** [comment]
> @aquarat I just tested on my windows machine and it works (I deployed on AWS+rinkeby with Airnode examples). The only change I needed to make was disable the line below in deploy-airnode script (as @dcroote pointed out in some jira issue):
> 
> ```
> // `-e USER_ID=$(id -u) -e GROUP_ID=$(id -g)`,
> ```
> 
> can you update that as well? (you can use similar check to https://github.com/api3dao/airnode/blob/master/packages/airnode-examples/scripts/create-airnode-secrets.ts#L6).
> 
> If someone could test GCP that would be nice (maybe Martin K.)

Thanks @Siegrift, I'll do this now.

**Comment by aquarat** [comment]
I am currently testing the latest commit.

File: packages/airnode-examples/scripts/remove-airnode.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ const main = async () => {
   const secretsFilePath = join(__dirname, '../aws.env');
   const deployCommand = [
     `docker run -it --rm`,
-    `-e USER_ID=$(id -u) -e GROUP_ID=$(id -g)`,
+    isMacOrWindows() ? '' : `-e USER_ID=$(id -u) -e GROUP_ID=$(id -g)`,
```
[the remark this answers is not in the document]
Comment (aquarat) [reply packages/airnode-examples/scripts/remove-airnode.ts:15]: You're probably right. I will test and advise. This change works on Windows and Linux.

**Comment by dcroote** [comment]
Success on `airnode-examples` with cmd! I did run into an unrelated issue with `nodeVersion` when removing though: 
```sh
Error: Invalid Airnode receipt file: [{"message":"deployment.nodeVersion: nodeVersion should be set to 0.4.*","level":"error"},{"level":"warning","message":"Extra field: deployment.cloudProvider.disableConcurrencyReservations"}]
```

@Siegrift / @amarthadan  - I believe this is because `nodeVersion` is set in this line since the `v0.5` branch merge:
https://github.com/api3dao/airnode/blob/8bb8b9fb032e979454be0b536cafed6d6bc0de94/packages/airnode-examples/integrations/config-utils.ts#L49

**Comment by aquarat** [comment]
> @Siegrift / @amarthadan - I believe this is because `nodeVersion` is set in this line since the `v0.5` branch merge:

I also came across this, I assumed it was related to someone else's WIP.

**Review by dcroote — APPROVED** [review]

**Comment by aquarat** [comment]
This works for me on Windows and Linux. I haven't tested on MacOS, yet.

**Comment by aquarat** [comment]
Just tested on MacOS M1 and it also works with these changes (all OSes work except for the `receipt.json` version mismatch).

**Pull request merged**